.github/workflows/terraform-daily-healthcheck-deploy-and-destroy.yml

```diff
@@ -1,90 +1,186 @@
-name: 'Daily Healthcheck Deploy and Destroy'
+name: 'Daily Healthcheck Deploy, Test and Destroy'
 
-#on:
-#  schedule:
-#    - cron: 45 7 * * 1-5
 on:
-  workflow_dispatch:
-    inputs:
-      buildBranch:
-        description: 'Feature branch to push to sandbox.'
-        required: true
-        type: 'string'
-      sandboxWorkspace:
-        description: 'Which Sandbox to push to.'
-        required: true
-        type: 'string'
-      environment:
-        default: 'development'
-        description: 'Which environment should this run against'
-        required: true
-        type: 'string'
-#
-#permissions:
-#  pull-requests: write
-#  id-token: write # This is required for requesting the JWT
-#  contents: read # This is required for actions/checkout
+ schedule:
+   - cron: 30 4 * * 1-5
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
 
 jobs:
-  test-job:
+  terraform_process:
     runs-on: ubuntu-latest
+    environment: development
+
+    steps:
+      # Checkout the repository to the GitHub Actions runner
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE }}
+          role-skip-session-tagging: true
+          aws-region: ${{ vars.AWS_REGION }}
+          mask-aws-account-id: true
+
+      - name: View AWS Role
+        run: aws sts get-caller-identity
+
+      # Install the latest version of Terraform CLI and configure the Terraform CLI configuration file with a Terraform Cloud user API token
```

Comment:
Would it be ok to remove these comments (and the ones below)? I think the config steps are quite self explanatory for anyone who is able to read the yml